frontend/packages/local-storage-operator-plugin/src/components/disks-list/disks-list-page.tsx

```diff
@@ -131,16 +159,17 @@ export const NodesDisksListPage: React.FC<NodesDisksListPageProps> = ({
         <>
```

Comment:
We are passing `subscriptionLoaded` in `loaded` customData itself, so not required any separate loading state.
```
     {!subscriptionLoaded ? (
        <LoadingInline />
) : (
```